Delivery-service/Entity/DeliverySystem/Program.cs

```diff
@@ -9,10 +10,11 @@ class Program
     {
         static void Main(string[] args)
```

Comment:
try to use classlib project type
not console app project type

Reply:
don't understand how to use classlib project type

Reply:
![image](https://user-images.githubusercontent.com/82231829/132251270-96a9032c-123f-4bcf-8f8e-25cc48b11948.png)

Reply:
![image](https://user-images.githubusercontent.com/82231829/132251306-9657a3c1-b940-4bc2-9917-14790be52e77.png)

Reply:
in properties